ignite/cmd/plugin.go

```diff
@@ -57,6 +61,97 @@ func UnloadPlugins() {
 	}
 }
 
+func linkPluginHooks(rootCmd *cobra.Command, p *plugin.Plugin) {
+	if p.Error != nil {
+		return
+	}
+
+	for _, hook := range p.Interface.Hooks() {
+		linkPluginHook(rootCmd, p, hook)
+	}
+}
+
+func linkPluginHook(rootCmd *cobra.Command, p *plugin.Plugin, hook plugin.Hook) {
+	cmdPath := hook.PlaceHookOn
+
+	if !strings.HasPrefix(cmdPath, "ignite") {
+		// cmdPath must start with `ignite ` before comparison with
+		// cmd.CommandPath()
+		cmdPath = "ignite " + cmdPath
```

Comment:
Let's refactor this "ignite " string literal so that it is a `const` declaration at the top of the file.  

```go
const (
igniteCmdPrefix = "ignite "
)
```